### PR TITLE
images: Fix long fedora-36 boot time, install qemu-kvm-core metapackage on Fedora

### PR DESCRIPTION
**Pull request by martinpitt**
The latest image refresh now takes very long to boot as it times out on
NetworkManager-wait-online.service.

Fedora 36 dropped support for the ancient network-scripts, which
previously disabled eth1 in NetworkManager. Replace the ignoring of eth1
with a NM configuration snippet, same as on Debian, Ubuntu, and Arch.

---

This was introduced in https://github.com/cockpit-project/bots/pull/3060 -- tests [work](https://logs.cockpit-project.org/logs/pull-3060-20220316-053526-dfa0fe77-fedora-36-cockpit-project-cockpit/log.html), but took 54 minutes instead of the usual 30, due to the very long boot time of each VM.

 * [x] image-refresh fedora-35
 * [x] image-refresh fedora-36

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh fedora-35 done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-35-20220316-095353

**Comment by cockpituous** [comment]
image-refresh in progress on centosci-tasks-v8j2m.
Log: https://logs-https-frontdoor.apps.ocp.ci.centos.org/logs/image-refresh-3099-20220316-095353/

**Comment by cockpituous** [comment]
image-refresh in progress on 2-ci-srv-04.
Log: https://logs.cockpit-project.org/logs/image-refresh-3099-20220316-095353/

**Comment by cockpituous** [comment]
image-refresh fedora-36 done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-36-20220316-101414

**Comment by cockpituous** [comment]
image-refresh in progress on centosci-tasks-nz7k4.
Log: https://logs-https-frontdoor.apps.ocp.ci.centos.org/logs/image-refresh-3099-20220316-101414/

**Comment by cockpituous** [comment]
Task failed: https://logs-https-frontdoor.apps.ocp.ci.centos.org/logs/image-refresh-3099-20220316-101414/

**Comment by martinpitt** [comment]
[there, 36 minutes](https://logs.cockpit-project.org/logs/pull-3099-20220316-103506-2512f635-fedora-36-cockpit-project-cockpit/log.html)

File: images/scripts/fedora.setup [hunk 1]
```diff
@@ -34,7 +34,7 @@ PackageKit \
 pcp \
 pcp-libs \
 python3-tracer \
-qemu-system-x86-core \
+qemu-kvm-core \
```
Comment (skobyda) [inline images/scripts/fedora.setup:37]: What is the motivation for this change in this specific image-refresh?

Comment (martinpitt) [reply images/scripts/fedora.setup:37]: It's unrelated to the boot time fix of course, but I'll have to do this change anyway. So instead of going through all fedora image refreshes twice, it's more efficient to just rebuild them once. It doesn't cause any regression, and unblocks the rpm dependency cleanup.

**Review by skobyda — APPROVED** [review]
